pkg/cli/cli.go

```diff
@@ -910,3 +925,52 @@ func buildServiceRootEndpoint(serviceURL, zone string) string {
 	const version = "v1"
 	return client.BuildRootEndpoint(serviceURL, version, zone)
 }
+
+type stateOutput struct {
+	// State description.
+	Message string
+	// True if state is completed.
+	Done bool
```

Comment:
nit: Consider having a `done` message instead of a flag in the state output. It could be something like `doneStateMsg = "\n"

Alternatively, add another method to `statePrinter` to print the "done" marker.

Both approaches avoid having to print the same message twice in lines 577 and 579.

Reply:
either in verbose logging runs or while not on an interactive console it's better two see the two different state entries. added a new PrintDone method to the API for better usability.